src/mina-signer/src/types.ts

```diff
@@ -9,7 +9,13 @@ export type Field = number | bigint | string;
 export type PublicKey = string;
 export type PrivateKey = string;
 export type Signature = SignatureJson;
-export type NetworkId = 'mainnet' | 'testnet';
+export type NetworkId = 'mainnet' | 'testnet' | { custom: string };
+
+export const NetworkId = {
```

Comment:
internal helper to get the plain `networkId`

Reply:
nicely done!